### DISCUSSION TITLE
Create a new job programmatically

### DISCUSSION THREAD

**Discussion by storny11** [quartznet/quartznet#1327] [Q&A] [answered]
Is it possible to schedule new jobs programmatically?

I have created one Job and now want to be able to schedule multiple instances of the same job with different schedules and different parameters. Is there a way to achieve that using Quartz?

**Comment by lahma** [comment, marked as answer]
Call `ScheduleJob` with the new triggers? Make sure you point your triggers to same job.

https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/jobs-and-triggers.html
